netbox/extras/management/commands/buildschema.py

```diff
@@ -0,0 +1,60 @@
+from os import path as os_path
+from json import dumps as json_dumps
+from json import loads as json_loads
+from jinja2 import FileSystemLoader, Environment
+
+from django.core.management.base import BaseCommand
+
+from dcim.choices import DeviceAirflowChoices
```

Comment:
I think it would be better to clean up the import like `from dcim.choices import *`

Reply:
I personally disagree with this. To avoid excess data load, aka imports, we should only import what is needed to improve performance. Yes it might be negligable at the end of the day, but any saved CPU cycles are better than nothing

Reply:
Avoiding the wildcard doesn't really save anything. The entire module must be loaded into memory either way; the explicit imports just limit what gets populated in the namespace.

Reply:
Okay, so would you prefer that we change this to `*`?